ddtrace/internal/datadog/profiling/stack_v2/include/stack_renderer.hpp

```diff
@@ -27,6 +42,8 @@ class StackRenderer : public RendererInterface
                                      uintptr_t thread_id,
                                      unsigned long native_id) override;
 
+    virtual void render_task_begin(std::string_view name);
+    virtual void render_pid_tid(uintptr_t pid, uintptr_t tid) override;
```

Comment:
```
/root/project/ddtrace/internal/datadog/profiling/stack_v2/include/stack_renderer.hpp:46:18: error: ‘virtual void Datadog::StackRenderer::render_pid_tid(uintptr_t, uintptr_t)’ marked ‘override’, but does not override
     virtual void render_pid_tid(uintptr_t pid, uintptr_t tid) override;
```

Reply:
Good catch--the `pid_tid` call was debris from a refactor.  I removed that function and propagated the changes.